pallets/vc-management/src/lib.rs

```diff
@@ -175,6 +192,10 @@ pub mod pallet {
 
 	#[pallet::error]
 	pub enum Error<T> {
+		/// a delegatee doesn't exist
+		DelegateeNotExist,
+		/// a `request_vc` request from unauthorised user
+		UnauthorisedUser,
```

Comment:
Q: are we using British or American spelling? :3

Reply:
Thanks, let's stick to American spelling - I'll check it (and other places)